<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hardware</class>
 <widget class="QWidget" name="Hardware">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>1419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="29" column="0">
    <widget class="QLabel" name="sqLabel">
     <property name="text">
      <string>SQ</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="AutoLineEdit" name="pot4Name">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="30" column="0">
    <widget class="QLabel" name="srLabel">
     <property name="text">
      <string>SR</string>
     </property>
    </widget>
   </item>
   <item row="27" column="1">
    <widget class="AutoLineEdit" name="soName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="29" column="1">
    <widget class="AutoLineEdit" name="sqName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="ls2Label">
     <property name="text">
      <string>LS2</string>
     </property>
    </widget>
   </item>
   <item row="23" column="2">
    <widget class="AutoComboBox" name="skType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="28" column="1">
    <widget class="AutoLineEdit" name="spName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="28" column="0">
    <widget class="QLabel" name="spLabel">
     <property name="text">
      <string>SP</string>
     </property>
    </widget>
   </item>
   <item row="27" column="0">
    <widget class="QLabel" name="soLabel">
     <property name="text">
      <string>SO</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="pot4Label">
     <property name="text">
      <string>S4</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="AutoLineEdit" name="pot3Name">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="rsLabel">
     <property name="text">
      <string>RS</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLabel" name="sbLabel">
     <property name="text">
      <string>SB</string>
     </property>
    </widget>
   </item>
   <item row="41" column="0">
    <widget class="QLabel" name="label_33">
     <property name="text">
      <string>PPM 2</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="AutoLineEdit" name="pot2Name">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="14" column="2">
    <widget class="AutoComboBox" name="sbType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="13" column="2">
    <widget class="AutoComboBox" name="saType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="34" column="1">
    <widget class="QComboBox" name="serialPortMode">
     <item>
      <property name="text">
       <string>OFF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>S-Port Mirror</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Telemetry</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SBUS Trainer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Debug</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="AutoLineEdit" name="sbName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="AutoLineEdit" name="pot1Name">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="AutoLineEdit" name="saName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="rudLabel">
     <property name="text">
      <string>Rud</string>
     </property>
    </widget>
   </item>
   <item row="37" column="1">
    <widget class="QSpinBox" name="txCurrentCalibration">
     <property name="minimum">
      <number>-49</number>
     </property>
     <property name="maximum">
      <number>49</number>
     </property>
    </widget>
   </item>
   <item row="42" column="0">
    <widget class="QLabel" name="label_34">
     <property name="text">
      <string>PPM 3</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="AutoLineEdit" name="rudName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="pot1Label">
     <property name="text">
      <string>S1</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="AutoLineEdit" name="eleName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="pot2Label">
     <property name="text">
      <string>S2</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="pot3Label">
     <property name="text">
      <string>S3</string>
     </property>
    </widget>
   </item>
   <item row="39" column="0">
    <widget class="QLabel" name="label_23">
     <property name="text">
      <string>PPM 1</string>
     </property>
    </widget>
   </item>
   <item row="39" column="1">
    <widget class="QSpinBox" name="PPM1">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="34" column="0" alignment="Qt::AlignLeft">
    <widget class="QLabel" name="serialPortLabel">
     <property name="text">
      <string>Serial Port</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="36" column="1">
    <widget class="QDoubleSpinBox" name="txVoltageCalibration">
     <property name="statusTip">
      <string/>
     </property>
     <property name="suffix">
      <string>v</string>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>-9.900000000000000</double>
     </property>
     <property name="maximum">
      <double>9.900000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="44" column="0">
    <widget class="QLabel" name="label_30">
     <property name="text">
      <string>PPM Multiplier</string>
     </property>
    </widget>
   </item>
   <item row="43" column="1">
    <widget class="QSpinBox" name="PPM4">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="24" column="2">
    <widget class="AutoComboBox" name="slType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="37" column="0">
    <widget class="QLabel" name="txCurrentCalibrationLabel">
     <property name="text">
      <string>Current Offset</string>
     </property>
    </widget>
   </item>
   <item row="43" column="0">
    <widget class="QLabel" name="label_35">
     <property name="text">
      <string>PPM 4</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="AutoLineEdit" name="ailName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="saLabel">
     <property name="text">
      <string>SA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="eleLabel">
     <property name="text">
      <string>Ele</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="AutoLineEdit" name="thrName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="AutoLineEdit" name="lsName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="ailLabel">
     <property name="text">
      <string>Ail</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="AutoLineEdit" name="rsName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="thrLabel">
     <property name="text">
      <string>Thr</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1">
    <widget class="AutoLineEdit" name="scName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="scLabel">
     <property name="text">
      <string>SC</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <widget class="AutoLineEdit" name="sdName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lsLabel">
     <property name="text">
      <string>LS</string>
     </property>
    </widget>
   </item>
   <item row="17" column="1">
    <widget class="AutoLineEdit" name="seName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="15" column="2">
    <widget class="AutoComboBox" name="scType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="18" column="1">
    <widget class="AutoLineEdit" name="sfName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="16" column="0">
    <widget class="QLabel" name="sdLabel">
     <property name="text">
      <string>SD</string>
     </property>
    </widget>
   </item>
   <item row="16" column="2">
    <widget class="AutoComboBox" name="sdType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="41" column="1">
    <widget class="QSpinBox" name="PPM2">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="36" column="0">
    <widget class="QLabel" name="txVoltageCalibrationLabel">
     <property name="text">
      <string>Battery Offset</string>
     </property>
    </widget>
   </item>
   <item row="42" column="1">
    <widget class="QSpinBox" name="PPM3">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="minimum">
      <number>-9999</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
   </item>
   <item row="19" column="1">
    <widget class="AutoLineEdit" name="sgName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="seLabel">
     <property name="text">
      <string>SE</string>
     </property>
    </widget>
   </item>
   <item row="17" column="2">
    <widget class="AutoComboBox" name="seType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="20" column="1">
    <widget class="AutoLineEdit" name="shName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <widget class="QLabel" name="sfLabel">
     <property name="text">
      <string>SF</string>
     </property>
    </widget>
   </item>
   <item row="18" column="2">
    <widget class="AutoComboBox" name="sfType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="21" column="1">
    <widget class="AutoLineEdit" name="siName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="19" column="0">
    <widget class="QLabel" name="sgLabel">
     <property name="text">
      <string>SG</string>
     </property>
    </widget>
   </item>
   <item row="19" column="2">
    <widget class="AutoComboBox" name="sgType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="22" column="1">
    <widget class="AutoLineEdit" name="sjName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="20" column="0">
    <widget class="QLabel" name="shLabel">
     <property name="text">
      <string>SH</string>
     </property>
    </widget>
   </item>
   <item row="23" column="1">
    <widget class="AutoLineEdit" name="skName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="21" column="0">
    <widget class="QLabel" name="siLabel">
     <property name="text">
      <string>SI</string>
     </property>
    </widget>
   </item>
   <item row="24" column="1">
    <widget class="AutoLineEdit" name="slName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="22" column="0">
    <widget class="QLabel" name="sjLabel">
     <property name="text">
      <string>SJ</string>
     </property>
    </widget>
   </item>
   <item row="25" column="1">
    <widget class="AutoLineEdit" name="smName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="23" column="0">
    <widget class="QLabel" name="skLabel">
     <property name="text">
      <string>SK</string>
     </property>
    </widget>
   </item>
   <item row="26" column="1">
    <widget class="AutoLineEdit" name="snName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="24" column="0">
    <widget class="QLabel" name="slLabel">
     <property name="text">
      <string>SL</string>
     </property>
    </widget>
   </item>
   <item row="25" column="0">
    <widget class="QLabel" name="smLabel">
     <property name="text">
      <string>SM</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="AutoLineEdit" name="rs2Name">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="26" column="0">
    <widget class="QLabel" name="snLabel">
     <property name="text">
      <string>SN</string>
     </property>
    </widget>
   </item>
   <item row="44" column="1">
    <widget class="QDoubleSpinBox" name="PPM_MultiplierDSB">
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="maximum">
      <double>5.000000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="AutoLineEdit" name="ls2Name">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="rs2Label">
     <property name="text">
      <string>RS2</string>
     </property>
    </widget>
   </item>
   <item row="20" column="2">
    <widget class="AutoComboBox" name="shType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="21" column="2">
    <widget class="AutoComboBox" name="siType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="22" column="2">
    <widget class="AutoComboBox" name="sjType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="25" column="2">
    <widget class="AutoComboBox" name="smType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="26" column="2">
    <widget class="AutoComboBox" name="snType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="27" column="2">
    <widget class="AutoComboBox" name="soType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="28" column="2">
    <widget class="AutoComboBox" name="spType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="29" column="2">
    <widget class="AutoComboBox" name="sqType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="30" column="2">
    <widget class="AutoComboBox" name="srType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="AutoComboBox" name="rs2Type">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="AutoComboBox" name="ls2Type">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="AutoComboBox" name="rsType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="AutoComboBox" name="lsType">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="AutoComboBox" name="pot4Type">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="AutoComboBox" name="pot3Type">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="AutoComboBox" name="pot2Type">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="AutoComboBox" name="pot1Type">
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
    </widget>
   </item>
   <item row="32" column="0">
    <widget class="QLabel" name="bluetoothLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Bluetooth</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="30" column="1">
    <widget class="AutoLineEdit" name="srName">
     <property name="maxLength">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="33" column="1">
    <widget class="QCheckBox" name="filterEnable">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="33" column="0">
    <widget class="QLabel" name="filterLabel">
     <property name="text">
      <string>ADC Filter</string>
     </property>
    </widget>
   </item>
   <item row="32" column="1">
    <widget class="QWidget" name="bluetoothWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="AutoComboBox" name="bluetoothMode">
        <property name="text" stdset="0">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="btNameLabel">
        <property name="text">
         <string>Device Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="AutoLineEdit" name="bluetoothName">
        <property name="maxLength">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="31" column="0" colspan="3">
    <widget class="Line" name="potsTypeSeparator_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="38" column="0" colspan="3">
    <widget class="Line" name="ppmSeparator">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="Line" name="potsTypeSeparator_1">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>10</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AutoComboBox</class>
   <extends>QComboBox</extends>
   <header>autocombobox.h</header>
  </customwidget>
  <customwidget>
   <class>AutoLineEdit</class>
   <extends>QLineEdit</extends>
   <header>autolineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
